### DISCUSSION TITLE
How can I submit a typo correction?

### DISCUSSION THREAD

**Discussion by joaogomes1** [freeplane/freeplane#1270] [Help] [answered]
Hi,
I have Freeplane 1.11.4.
I am using Portuguese language.
I've found a typo in one of the menus.
I've attached a screenshot with two marked words, where the first 'z' should be replaced with 's'.
So, instead of 'vizualização', the correct would be 'visualização'.
![Captura de tela 2023-06-28 002937](https://github.com/freeplane/freeplane/assets/86538386/ee41bc85-448f-4518-9b0f-c998b9b2ca7b)

**Comment by ldmpub** [comment, marked as answer]
Hi,

You can perform the change by yourself on weblate (Freeplane is an open source project and translation do not need any technical skills) see weblate.org.

**Reply by ldmpub** [reply]
More precisely => https://hosted.weblate.org/projects/weblate/application/pt/ 

Note: you need first to create an account
